cmd/local/conf/conf_test.go

```diff
@@ -341,3 +341,12 @@ func TestConfReadingWhenLoggingParsingOfDdcYAML(t *testing.T) {
 	}
 	afterEachConfTest()
 }
+
+func TestURLsuffix(t *testing.T) {
```

Comment:
add a test that has not trailing slash. need to verify we don't break anything (I know we don't but it's a good regression test)

Reply:
Ok this is added into the same function - lmk if this looks good

Reply:
that looks wrong ..expected shouldn't _add_ a / should it?

```
	testURL = "http://localhost:9047/some/path"
 	expected = "http://localhost:9047/some/path/"
 	actual = conf.SanitiseURL(testURL)
 	if expected != actual {
 		t.Errorf("\nexpected: %v\nactual: %v\n'", expected, actual)
```

Reply:
sorry - typo 🤦